assets/sass/application.scss

```diff
@@ -7,11 +7,13 @@ $govuk-page-width: $moj-page-width;
 @import 'govuk/all';
 @import 'moj/all';
 
+@import 'node_modules/@ministryofjustice/hmpps-connect-dps-components/dist/assets/footer';
+// TODO enable: @import 'node_modules/@ministryofjustice/hmpps-connect-dps-components/dist/assets/header-bar';
```

Comment:
Have left the various header-related bits from following the set-up instructions temporarily commented out. Will be turning on in following PRs...